packages/namekit-react/src/components/Tooltip.tsx

```diff
@@ -12,19 +12,21 @@ type Props = {
 export const DEFAULT_MAX_TOOLTIP_WIDTH = 400;
 
 export const TooltipPlacement = {
-  TOP: "top",
```

Comment:
We shouldn't write enum values this way. We should always follow the same standard for how we write them.